go/install/install_windows.go

```diff
@@ -374,7 +374,16 @@ func LsofMount(mountDir string, log Log) ([]CommonLsofResult, error) {
 	return nil, fmt.Errorf("Cannot use lsof on Windows.")
 }
 
+// delete this function and calls to it if present after 2022
+func deleteDeprecatedFileIfPresent() {
+	// this file is no longer how we do things, and if it's present (which it shouldn't be) it could
+	// cause unexpected behavior
+	autostartLinkPath := filepath.Join(appDataDir, "Microsoft\\Windows\\Start Menu\\Programs\\Startup\\KeybaseStartup.lnk")
+	_ = os.Remove(autostartLinkPath)
```

Comment:
Just to make sure appDataDir ends in '\'?

Reply:
shouldn't matter `filepath.Join` looks smart enough to deal with it either way. 
and i made this code now matches almost exactly the existing code in `deprecatedStartupInfo`